src/components/Ranking.js

```diff
@@ -15,6 +15,7 @@ const Ranking = () => {
     });
   }, []);
 
+  // TODO: DISPLAY SORTED OPTIONS
```

Comment:
Hm, it seems you still have some work to do here. But nice, I see is also your first PR, keep up good work!

Reply:
This is just a draft for now.. but thanks for reviewing it, will ping you when it's done

Reply:
Ups, i didn't see it! Okay okay, my bad 😅